### ISSUE TITLE
About semantic segmentation on Scannet validation set

### ISSUE THREAD

**Issue by Asterisci** [fcendra/sl3d#1]
Thanks for your great work!
It seems there are no semantic segmentation results on the raw Scannet validation set in the paper. Have you done any experiments on it? Or can you provide the pretrained weights for evaluation?
